### PR TITLE
Update to Swift 4, project settings, and fix deprecated AlertView.

### PR DESCRIPTION
**Pull request by djensenius**

### CODE REVIEW

File: Processing for iOS/ProjectSelectionTableViewController.swift [hunk 1]
```diff
@@ -164,25 +164,22 @@ class ProjectSelectionTableViewController: UITableViewController, UIViewControll
...
                 
                 var message = ""
                 var suggestedName = fileName
                 if fileName == "" {
                     message = "Name should be at least one character"
-                } else if nameAlreadyExists(name: fileName) {
+                } else if self.nameAlreadyExists(name: fileName) {
                     message = "File with name '\(fileName)' already exists. Please choose another name or delete the exiting one first."
                 } else if fileName.contains(" ") {
                     message = "File name should not contain spaces."
```
Comment (djensenius) [inline Processing for iOS/ProjectSelectionTableViewController.swift]: I guess this needs to be the case because Processing?

Comment (frogg) [reply Processing for iOS/ProjectSelectionTableViewController.swift]: The reason why the name should not contain spaces? Yeah, the reason was to not c confuse the folder structure of Processing.

File: Processing for iOS/ProjectSelectionTableViewController.swift [hunk 2]
```diff
@@ -217,15 +214,21 @@ class ProjectSelectionTableViewController: UITableViewController, UIViewControll
...
-                    alertView.textField(at: 0)?.text = suggestedName
-                }
-                alertView.show()
-                
+                self.showCreateAlert(title: message, name: suggestedName)
             }
-        }
+        }))
+        
+        alertController.addAction(UIAlertAction(title: "Cancel", style: .default
```
Comment (frogg) [inline Processing for iOS/ProjectSelectionTableViewController.swift]: The actions should be added the other way around. The "create" button is usually on the right side. Also, it should get the `.cancel` style.

File: Processing for iOS/ProjectSelectionTableViewController.swift [hunk 3]
```diff
@@ -217,15 +214,21 @@ class ProjectSelectionTableViewController: UITableViewController, UIViewControll
...
+            , handler: { (_) in
+                alertController.dismiss(animated: true, completion: nil)
+        }))
+        
+        alertController.addTextField(configurationHandler: { (textField) -> Void in
+            textField.textAlignment = .center
```
Comment (frogg) [inline Processing for iOS/ProjectSelectionTableViewController.swift]: Any reason to make it `.center`? I think normally I've seen it `.left` only.

**Pull request merged**